content/Circular_Intersection.md

```diff
@@ -0,0 +1,59 @@
+---
+title: Circular Intersections
+draft: true
+tags:
+  - math
+---
+# Circular Intersections
+## How I started this problem
+One day I was writing my school HW where I had to fill out a venn diagram contrasting to people. Because the middle section soon ran out of space, I thought: *How can I make all three sections the same?*
```

Comment:
`Because the middle section soon ran out of space, I thought: *How can I make all three sections the same?*`: this is also hard to reason about. the first sentence (situation) does not naturally leads to the second sentence (question).  maybe need more words here to make your thought process more accessible to your readers.